src/mpyl/cli/commands/build/mpyl.py

```diff
@@ -126,9 +116,7 @@ def run_mpyl(
                 steps = Steps(
                     logger=logger,
                     properties=mpyl_run_parameters.run_config.run_properties,
-                    steps_collection=StepsCollection(
-                        logger=logger, base_path="src" if params.local else None
-                    ),
+                    steps_collection=StepsCollection(logger=logger),
```

Comment:
We needed this change with Jorg to run mpyl locally. Same idea as the imports in the runner where we check if it's running locally or not. We probably have different local setup then?